packages/components/button/src/IconButton/README.mdx

```diff
@@ -5,33 +5,44 @@ status: 'stable'
 section: 'buttonComponents'
 slug: /components/icon-button/
 github: 'https://github.com/contentful/forma-36/tree/forma-v4/packages/components/button/src/IconButton'
-storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/components-button-iconbutton--basic'
 typescript: ./IconButton.tsx
 ---
 
-import { Props } from '@contentful/f36-docs-utils';
-import { PreviewIcon } from '@contentful/f36-icons';
+`IconButton` composes the [Button](../components/button) component except that it renders only an icon.
 
-## How to use IconButton
+## Import
 
-IconButton composes the Button component except that it renders only an icon.
-You must pass the Icon component you want to render to the `icon` prop, you may also provide the variant and size to the Icon.
+```js static=true
+import { IconButton } from '@contentful/f36-components';
+```
 
-## Code examples
+## Examples
 
-Example of simple usage:
+### Sizes and Variations
 
-```jsx
-// import { IconButton } from '@contentful/f36-button';
-// import { PreviewIcon } from '@contentful/f36-icons';
+You need to pass an [Icon component](../components/icon) you want to render to the `icon` prop, you may also provide `variant` and `size` to the icon.
+
+```jsx file=../../examples/IconButtonExample.tsx
+
+```
+
+### As toolbar actions
+
+Usually `IconButton` should be used in a toolbars, action bars and in all other space restricted containers where rendering `Button` is not possible.
```

Comment:
it suggests that this is the most common use case. I don't think that's true. Maybe we could do something like:

"In toolbars, action bars, and in all other space restricted containers we suggest using `IconButton` component. For those cases, it would be also beneficial to use [Tooltip](../components/tooltip) as a wrapper of `IconButton` to help users with understanding the button's purpose."

Reply:
A great suggestion

Reply:
done